build/yaml/steps/windows/dotnettest.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Step Template: Test Windows Binaries
+
+parameters:
+  SubFolder: ''
+  RunTitle: ''
+
+steps:
+- task: DotNetCoreCLI@2
+  displayName: 'Run Tests: $(Platform)\${{ parameters.SubFolder }}*Tests*.dll'
+  inputs:
+    command: test
+    arguments: >-
```

Comment:
>>- [](http://example.com/codeflow?start=15&length=2)

Is this yaml syntax or an argument for dotnet test?

Reply:
This yaml syntax says that each item in the following section should be interpreted as a concatenation of each together using a single space between each element to produce a single string. It allows you to break apart really long strings so it's more readable/diff-able.

Reply:
awesome, learned something new!